src/keysBuilder/extractTSGetTextMarkerKeys.ts

```diff
@@ -0,0 +1,30 @@
+import { tsquery } from '@phenomnomnominal/tsquery';
+import { buildKeysFromASTNodes } from './buildKeysFromASTNodes';
+
+export function extractGetTextMarkerKeys(ast): { key: string; lang: string }[] {
+  const importNodes = tsquery(ast, `ImportDeclaration:has([text=@ngneat/transloco-keys-manager])`);
+  if (importNodes[0]) {
+    const alias = getAlias(importNodes[0]);
+    const fns = tsquery(importNodes[0].parent, `CallExpression Identifier[text=${alias || 'getText'}]`);
+    return buildKeysFromASTNodes(fns, [alias || 'getText']);
+  }
+  return [];
+}
+
+function getAlias(nodes: any) {
```

Comment:
rename to `getMarkerName`, `nodes` => `importNode: ts.Node`.
I'm not sure whats all the logic you did here, but you can simply do:
```typescript
const [defaultName, alias] = tsquery(importNode, 'ImportSpecifier Identifier');

return (alias || defaultName).getText();
```

Reply:
done